src/graphics.rs

```diff
@@ -8,13 +8,17 @@ use crate::{
 use {
     anyhow::*,
     derivative::*,
+    image::Rgba,
```

Comment:
stylistically i prefer to have subsequent imports like these rolled into subtrees of the import statement, like `image::{Rgba, RgbaImage}`.